metadata-ingestion/src/datahub/ingestion/source/aws/glue.py

```diff
@@ -833,10 +833,8 @@ def get_profile_if_enabled(
                 **{k: v for k, v in kwargs.items() if v}
             )
 
-            partition_keys = response["Table"]["PartitionKeys"]
-
             # check if this table is partitioned
-            if partition_keys:
+            if partition_keys := response["Table"].get("PartitionKeys"):
```

Comment:
we can't use the `:=` walrus operator since we need to maintain compatibility with python 3.7

Reply:
got it will change